content/02.introduction.md

```diff
@@ -114,8 +114,18 @@ Respiratory failure remains the leading cause of death for COVID-19 patients [@d
 Besides major pulmonary damage, SARS-CoV-2 infection can damage many other organ systems including the heart [@doi:10.1001/jamacardio.2020.0950], kidney [@doi:10.1038/s41585-020-0319-7; @doi:10.1186/s13054-020-02872-z], liver [@doi:10/ggpx6s], and gastrointestinal tract [@doi:10.1053/j.gastro.2020.02.055; @doi:10.1111/1751-2980.12851].
 It can even cause neurological complications [@doi:10.1016/j.bbi.2020.03.031]. 
 As it becomes clearer that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.
-To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues.
-
+To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues/cells and also to measure the host transcriptional response after virus infection. 
+Below we summarize the existing transcriptome studies published after a peer-reviewed process as of May 2020 [Note to self (Vikas): change the date after some pre-prints are published]. 
+Although we tried to cover all appropriate studies, we realize we may have missed some.
+
+Two studies profiled expression response after SARS-CoV-2 infection in multiple human cell lines (Figure X1) [@pmid:32416070; @doi:10.1101/2020.05.05.079194]. 
+First study by Blanco-Melo et al performed in-depth analysis of the transcriptional response to SARS-CoV-2 compared with other respiratory viruses, including MERS-CoV, SARS-CoV-1, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV). 
+In addition to human cell lines A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands) and MRC-5 (human fetal lung fibroblast cells), authors measured the host transcriptional response to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), in nasal washes of ferrets as an animal model and in lung samples of two COVID-19 patients. 
+Differential expression analysis between infected and mock cell conditions revealed that the transcriptional response of the cells with efficient SARS-CoV-2 replication is significantly different from the host response of all other viruses tested. 
+A unique proinflammatory cytokine signature is present in both high and low multiplicity of infections (MOI). 
```

Comment:
Perhaps add an explanation of why MOI is interesting. I don't think we've introduced that concept in the text yet. I read the discussion of this paper, and it seems very interesting!

Reply:
Agree, I want to include this after second paper is introduced because they are using different MOI.